Source/Core/Core/NetPlayQoS.h

```diff
@@ -0,0 +1,23 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+struct _ENetPeer;
```

Comment:
In this case you may want to include the header. This is forward declaring an internal name in the enet library.